sources/src/main/java/com/google/solutions/jitaccess/web/ApiResource.java

```diff
@@ -66,7 +67,10 @@ public class ApiResource {
   ActivationTokenService activationTokenService;
 
   @Inject
-  NotificationService notificationService;
+  NotificationService<String> slackNotificationService;
+
+  @Inject
+  NotificationService<Notification> notificationService;
```

Comment:
Shouldn't a single (non-generic) instance of NotificationService suffice now?

Reply:
yepp, adjusted 🙌